src/main/res/values/strings.xml

```diff
@@ -749,33 +749,33 @@
 
     <!-- 2016-04-03 -->
     <string name="error_tor_not_installed">Tor will not work unless Orbot is installed. Install now?</string>
-    <string name="error_tor_setting_failed">Failed to set proxy.</string>
+    <string name="error_tor_setting_failed">Could not set proxy.</string>
 
     <!-- 2016-04-10 -->
     <string name="error_parse_imgur_title">Parse Error</string>
-    <string name="error_parse_imgur_message">Imgur sent some data I couldn\'t understand.</string>
+    <string name="error_parse_imgur_message">Imgur sent incomprehensible data.</string>
 
     <string name="error_upload_fail_imgur_title">Upload failed</string>
-    <string name="error_upload_fail_imgur_message">The upload was rejected by Imgur for an unknown reason.</string>
+    <string name="error_upload_fail_imgur_message">Rejected by Imgur for unknown reason.</string>
 
     <string name="upload_to_imgur">Upload to Imgur</string>
     <string name="no_file_selected">No file selected</string>
     <string name="button_upload">Upload</string>
     <string name="button_browse">Browse…</string>
 
     <string name="error_file_too_big_title">File too big</string>
-    <string name="error_file_too_big_message">The file size (%1$s) is larger than the upload limit (%2$s).</string>
+    <string name="error_file_too_big_message">Ensure the file size (%1$s) is smaller than the upload limit (%2$s).</string>
```

Comment:
Preferred original

Reply:
All the others are invocations to do something. Why then should this be just information that the user is meant to figure out?

Reply:
The new wording sounds slightly odd due to the numbers in the string, in the same way that "please ensure 5 is less than 3" doesn't make sense. I think it's generally just clearer as it was.

Reply:
Well if there isn't a denomination it doesn't make sense in either case. Should I add a "MB"?

Reply:
The denomination is added programmatically in `ImgurUploadActivity`